src/components/Cards/CardLarge/CardLarge.scss

```diff
@@ -0,0 +1,68 @@
+@import './../../../styles/utilities/colors.scss';
+@import './../../../styles/fonts.scss';
+@import './../../../styles/utilities/typography.scss';
+
+$card-image-height: 330px;
+
+/* stylelint-disable */
```

Comment:
You can disable this, all classes pass but there are a few syntax and property order fixes. Please do for all the other card scss files.

Reply:
Linter errors have been fixed.